src/utils/api.js

```diff
@@ -194,3 +195,27 @@ export const getOrgUnitGroupSets = () =>
         paging: false,
         fields: 'id,displayName',
     })
+
+/**
+ * @returns {Promise}
+ */
+export const getResources = (page, pageSize, search) => {
+    const requestData = {
+        page,
+        pageSize,
+        fields: 'displayName,id,url,external,access',
+        ...(!search
+            ? {}
+            : {
+                  filter: `displayName:ilike:${search}`,
+              }),
+    }
+
```

Comment:
Instead of using very complex syntax to optionally add a property to requestData, why not just do this here:
```javascript
if (search) {
    requestData.filter = `displayName:ilike:${search}`
}
```
(and get rid of what is now lines 207-211)